### PR TITLE
made changes to README.md for installations steps

### PR DESCRIPTION
**Pull request by hambali999**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,6 +67,25 @@ export const ExampleComponent = () => {
 
 </details>
 
+## Installation
```
Comment (igeligel) [inline README.md:70]: Could you please put the installation section before the usage section?

File: README.md [hunk 2]
```diff
@@ -67,6 +67,25 @@ export const ExampleComponent = () => {
...
+
+<details>
```
Comment (igeligel) [inline README.md:72]: I actually wanted two `details` section. One only for npm and one for yarn.

Should look like [this gist](https://gist.github.com/igeligel/7fb12f8808c15f777d66e03df6a074cd).

Comment (igeligel) [inline README.md:72]: Maybe I have not it stated it good enough in the ticket. I wanted to have two `details` section. One for yarn and one for npm.

Should look like [this gist](https://gist.github.com/igeligel/7fb12f8808c15f777d66e03df6a074cd).

**Review by igeligel — CHANGES_REQUESTED** [review]
Can you delete the package-lock.json as well, please? 

We are using yarn for development and do not support npm for development, see the _[development section in the README](https://github.com/igeligel/react-in-out-textarea#development)_.

**Comment by hambali999** [comment]
Hi @igeligel I have made the changes, please do review it

**Comment by igeligel** [comment]
@all-contributors please add @hambali999 for code

**Pull request merged**